README.md

```diff
@@ -56,7 +56,8 @@ Options:
   --owner, -o       GitHub owner of the PR repository                   [string]
   --repo, -r        GitHub repository of the PR                         [string]
   --file, -f        File to write the metadata in                       [string]
-  --check-comments  Check for 'LGTM' in comments                        [boolean]
+  --check-comments  Check for 'LGTM' in comments                       [boolean]
+  --warn-commits    Number of commits to warn              [number] [default: 3]
```

Comment:
On a second thought, maybe `--max-commits` otherwise this sounds like a boolean to turn on/off commit warning.

Reply:
Using `max-commits` flag now.